frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/DistrictView/index.test.tsx

```diff
@@ -3,7 +3,7 @@ import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
```

Comment:
I think the views (District, Forecast, Hometable, Timeline...) will be different for AA storm so they should probably live under AA Drought? What do you think?

Reply:
I guess we can move it later, I just want to be careful when copying components like this not to carry over extra complexity and avoidable dependence

Reply:
yes, I have not yet touched on the distinction between the view elements specific to each AA. Also i wanted to test if everything was loaded correctly when i switch AA.

I'm going to see what can be put in common for the two AA types, perhaps extracting common parts. 
But I understand that the current view elements are specific to AA drought and must be separated from AA storm

Reply:
I have moved the AA drought specific components to AADroughtPanel folder